### PR TITLE
Yue Hang Leung parameters for 26p5GeV_fixedTarget_2018 TpcRS parameters

### PR DESCRIPTION
**Pull request by fisyakbnl**

### CODE REVIEW

**Comment by fisyakbnl** [comment]
Accept

File: StarDb/Calibrations/tpc/TpcResponseSimulator.20180605.071409.C [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+//
+// Revision 1.3  2010/04/16 19:31:19  fisyak
+// Intermidiate version
+//
+// Revision 1.3  2010/04/04 23:14:33  fisyak
+// Add Row Correction
+//
+// Revision 1.2  2010/04/01 22:17:57  fisyak
+// Freeze version W
+//
```
Comment (plexoos) [inline StarDb/Calibrations/tpc/TpcResponseSimulator.20180605.071409.C:68]:
```suggestion
```

**Review by iraklic — APPROVED** [review]

**Comment by iraklic** [comment]
I just approved this file merging. I am not sure what @plexoos comment means? Do you suggest removing the historical comments that are in the file or is there something else?

**Comment by iraklic** [comment]
Also it looks like Yuri @yfisyak has two github accounts (another is @fisyakbnl) but author is only one. Looks like you need to approve it from @yfisyak to get it merged.

**Comment by veprbl** [comment]
> I just approved this file merging. I am not sure what @plexoos comment means? Do you suggest removing the historical comments that are in the file or is there something else?

Remove the (seemingly outdated?) comments

**Pull request closed without merging**